src/commands/stands/pectra.ts

```diff
@@ -67,7 +67,11 @@ export const PectraDevNetUp = command.cli({
     logger.log("✅ Lido Core protocol activated.");
 
     logger.log("🚀 Activating CSM protocol...");
-    await dre.runCommand(ActivateCSM, {});
+    await dre.runCommand(ActivateCSM, {
```

Comment:
We should return this line, because protocol deploy without this command is incomplete